docs/src/polynomial.md

```diff
@@ -1084,4 +1084,8 @@ U, z = polynomial_ring(R, "z")
 h = rand(U, 3:3, -1:2, -10:10)
 ```
 
+### Ring homomorphisms
 
+```@docs
+hom(::PolyRing, ::PolyRing, ::Any, ::RingElem)
```

Comment:
```suggestion
hom(::PolyRing, ::NCRing, ::Any, ::Any)
```
This is the new docstring you add to the function on line 207

Reply:
The docstring filtering is buggy nonsense, so if it works it works.